docs/editor/profiles.md

```diff
@@ -0,0 +1,111 @@
+---
+Order: 15
+Area: editor
+TOCTitle: Profiles
+ContentId: a65efc48-5a2d-4c7d-bd23-03f0393b53f6
+PageTitle: Profiles in Visual Studio Code
+DateApproved: 2/2/2023
+MetaDescription: Expand your development workflow with task integration in Visual Studio Code.
+---
+# Profiles in Visual Studio Code
+
+Visual Studio Code has 100's of [settings](/docs/getstarted/settings.md), 1000's of [extensions](/docs/editor/extension-marketplace.md), and unnumerable ways to adjust the layout and sizing of the UI to let you customize the editor. Profiles let you create sets of customizations and quickly switch between them or share them with others. This topic explains how to create, modify, export, and import Profiles.
+
+## Create a Profile
+
+VS Code treats your current configuration as the **Default Profile**. As you modify settings, install extensions, or open and close views and panels, these customization are tracked in the Default Profile.
+
+To create a new profile, you can use the **File** > **Preferences** > **Profiles** > **Create Profile** menu item (**Code** > > **Preferences** > **Profiles** > **Create Profile** on macOS).
+
+![Create Profile command](images/profiles/create-profile.png)
+
+You can create a new profile based on the current profile (**Profiles: Create from Current Profiles**) or create an Empty Profile. An Empty Profile includes no user customizations (settings, extensions, snippets, etc.) but with the UI state (view layouts and sizing) preserved.
```

Comment:
Empty profile also resets the UI state including view layout. Workspace state is not included which is sizes, editor layout etc., My suggestion would be not to separate UI state here. In FaQ or advanced section, we can clarrify this.